src/components/BookCard/BookCardElements.js

```diff
@@ -0,0 +1,226 @@
+import styled from 'styled-components';
+
+export const BookContainer = styled.div`
+        width: 100%;
+        max-width: 1100px;
+        margin: auto;
+        
+`;
+export const BookCardWrapper = styled.div`
+        width: 75rem;
+        height: 10.625rem;
```

Comment:
- [ ]  The same here also.

Reply:
Hi @dasileker,

Do you mean I can not use styled-components which is like sass in my react project?

Reply:
it would be better for your project to show different skills, I'm not saying that you can not but it's better.

Reply:
Great! This will be looked into.
But do you think the project should be disapproved because of this?